static/src/js/components/AdminPreferencesMetrics/__tests__/AdminPreferencesMetricsList.test.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import '@testing-library/jest-dom'
```

Comment:
If you import this into the jest setup file you just have to import it the one time and not into every test file that needs it

Reply:
This should probably be it's own ticket (1 point)